src/System.Management.Automation/engine/CoreAdapter.cs

```diff
@@ -4252,7 +4252,7 @@ protected override T GetMember<T>(object obj, string memberName)
             // are ignored.
             if (typeof(T) == typeof(PSMemberInfo))
             {
-                T returnValue = PSObject.dotNetInstanceAdapter.GetDotNetMethod<T>(obj, memberName);
+                T returnValue = base.GetDotNetMethod<T>(obj, memberName);
```

Comment:
Shouldn't this also be changed at line 4607?  It seems like GetDotNetMethod at least should be a static method, and a static instance is not needed.

Reply:
Yes, I think the use of `PSObject.dotNetInstanceAdapter` in `PropertyOnlyAdapter` can be changed to `base` too. `PropertyOnlyAdapter` derives from `DotNetAdapter`.

Reply:
I have made the similar change to `PropertyOnlyAdapter`.